### ISSUE TITLE
GroupFolders 15.1.0 breaking cron.php on NC 27.0.2

### ISSUE THREAD

**Issue by dzidek23** [nextcloud/groupfolders#2566]
**Issues fixed with the GroupFolder 15.2.0 - or at least that what it seems**
### Steps to reproduce
1. Running crontab as www-data with `*/5 * * * * php -f /var/www/nextcloud/cron.php`

### Expected behaviour
Should trigger NC jobs etc.

### Actual behaviour
```sudo -u www-data php -f /var/www/nextcloud/cron.php
Error: Interface "OCA\Files_Versions\Versions\INeedSyncVersionBackend" not found in /var/www/nextcloud/apps/groupfolders/lib/Versions/VersionsBackend.php:46
Stack trace:
#0 /var/www/nextcloud/lib/composer/composer/ClassLoader.php(582): include()
#1 /var/www/nextcloud/lib/composer/composer/ClassLoader.php(433): Composer\Autoload\{closure}()
#2 /var/www/nextcloud/apps/groupfolders/lib/AppInfo/Application.php(138): Composer\Autoload\ClassLoader->loadClass()
#3 /var/www/nextcloud/lib/private/AppFramework/Utility/SimpleContainer.php(171): OCA\GroupFolders\AppInfo\Application->OCA\GroupFolders\AppInfo\{closure}()
#4 /var/www/nextcloud/3rdparty/pimple/pimple/src/Pimple/Container.php(122): OC\AppFramework\Utility\SimpleContainer->OC\AppFramework\Utility\{closure}()
#5 /var/www/nextcloud/lib/private/AppFramework/Utility/SimpleContainer.php(138): Pimple\Container->offsetGet()
#6 /var/www/nextcloud/lib/private/AppFramework/DependencyInjection/DIContainer.php(488): OC\AppFramework\Utility\SimpleContainer->query()
#7 /var/www/nextcloud/lib/private/AppFramework/DependencyInjection/DIContainer.php(466): OC\AppFramework\DependencyInjection\DIContainer->queryNoFallback()
#8 /var/www/nextcloud/lib/private/AppFramework/Utility/SimpleContainer.php(97): OC\AppFramework\DependencyInjection\DIContainer->query()
#9 [internal function]: OC\AppFramework\Utility\SimpleContainer->OC\AppFramework\Utility\{closure}()
#10 /var/www/nextcloud/lib/private/AppFramework/Utility/SimpleContainer.php(84): array_map()
#11 /var/www/nextcloud/lib/private/AppFramework/Utility/SimpleContainer.php(124): OC\AppFramework\Utility\SimpleContainer->buildClass()
#12 /var/www/nextcloud/lib/private/AppFramework/Utility/SimpleContainer.php(142): OC\AppFramework\Utility\SimpleContainer->resolve()
#13 /var/www/nextcloud/lib/private/AppFramework/DependencyInjection/DIContainer.php(494): OC\AppFramework\Utility\SimpleContainer->query()
#14 /var/www/nextcloud/lib/private/AppFramework/DependencyInjection/DIContainer.php(466): OC\AppFramework\DependencyInjection\DIContainer->queryNoFallback()
#15 /var/www/nextcloud/lib/private/AppFramework/Utility/SimpleContainer.php(65): OC\AppFramework\DependencyInjection\DIContainer->query()
#16 /var/www/nextcloud/apps/groupfolders/lib/AppInfo/Application.php(183): OC\AppFramework\Utility\SimpleContainer->get()
#17 /var/www/nextcloud/lib/private/AppFramework/Utility/SimpleContainer.php(171): OCA\GroupFolders\AppInfo\Application->OCA\GroupFolders\AppInfo\{closure}()
#18 /var/www/nextcloud/3rdparty/pimple/pimple/src/Pimple/Container.php(122): OC\AppFramework\Utility\SimpleContainer->OC\AppFramework\Utility\{closure}()
#19 /var/www/nextcloud/lib/private/AppFramework/Utility/SimpleContainer.php(138): Pimple\Container->offsetGet()
#20 /var/www/nextcloud/lib/private/AppFramework/DependencyInjection/DIContainer.php(488): OC\AppFramework\Utility\SimpleContainer->query()
#21 /var/www/nextcloud/lib/private/ServerContainer.php(155): OC\AppFramework\DependencyInjection\DIContainer->queryNoFallback()
#22 /var/www/nextcloud/lib/private/AppFramework/Utility/SimpleContainer.php(65): OC\ServerContainer->query()
#23 /var/www/nextcloud/lib/public/Server.php(55): OC\AppFramework\Utility\SimpleContainer->get()
#24 /var/www/nextcloud/lib/private/BackgroundJob/JobList.php(318): OCP\Server::get()
#25 /var/www/nextcloud/lib/private/BackgroundJob/JobList.php(227): OC\BackgroundJob\JobList->buildJob()
#26 /var/www/nextcloud/cron.php(145): OC\BackgroundJob\JobList->getNext()
#27 {main}
```

### Server configuration

**Operating system:**
Debian 11
**Web server:**
Apache2 - 2.4.56
**Database:**
MariaDB 10.5
**PHP version:**
8.2
**Nextcloud version:** (see Nextcloud admin page)
Nextcloud Hub 5 (27.0.2)
**Group folders version:**
15.1.0
**Updated from an older Nextcloud/ownCloud or fresh install:**
Updated

**Comment by XueSheng-GIT** [comment]
> Issues fixed with the GroupFolder 15.2.0 - or at least that what it seems

What's the purpose of this issue if it is fixed already with the latest version?

**Comment by joshtrichards** [comment]
Fixed in #2549 
Released in v15.2.0 as noted
